samples/spring-data-jdbc/src/main/resources/create_schema.sql

```diff
@@ -0,0 +1,68 @@
+/*
+ This script creates the database schema for this sample application.
+ All lines that start with /* skip_on_open_source_pg */ are skipped when the application is running on a
+ normal PostgreSQL database. The same lines are executed when the application is running on a Cloud
+ Spanner database. The script is executed by the DatabaseSeeder class.
+*/
+
+create sequence if not exists id_generator
+/* skip_on_open_source_pg */ bit_reversed_positive
+;
+    
+create table if not exists singers (
+    id         bigint not null primary key default nextval('id_generator'),
+    first_name varchar,
+    last_name  varchar not null,
+    full_name  varchar generated always as (CASE WHEN first_name IS NULL THEN last_name
+                                                WHEN last_name  IS NULL THEN first_name
```

Comment:
last name is marked not null, so is this check required?

Reply:
Good point. Our `Singer` entity does allow `lastName` to be null, so I removed the `not null` constraint in this file and kept the check.